### ISSUE TITLE
possible filebot filename-matching bug

### ISSUE THREAD

**Issue by mickeldaelmans** [pannal/Sub-Zero.bundle#436]
I might have found a bug regarding the Filebot filename matching. I'll have to doublecheck if it's not an issue with the filemetadata when I get home.  

It seems to cut of at the **[** in the original filename (if it contains one); and this behaviour is visible in the log with all filenames that contain a [. filenames that do not contain a [ seem to be displayed fine in the logs. I guess either the file metadata isnt correct (will verify when I'm home), or maybe subzero doesnt grab the filename correctly (or another option: doesnt log it correctly).

**'bugged' match**

original filename:
``` 
A.Plastic.Ocean.2016.1080p.BluRay.x264-[YTS.AG].mp4
``` 
SZ-log:
```
2018-02-14 13:33:56,146 (7f8f404d3700) :  INFO (logger:38) - subliminal_patch.core: Refining video with filebot
2018-02-14 13:33:58,839 (7f8f404d3700) :  DEBUG (logger:28) - subliminal_patch.refiners.common: Got original filename: A.Plastic.Ocean.2016.1080p.BluRay.x264-
```
**'correct' match**

original filename:
``` 
the.alienist.s01e04.1080p.webrip.x264-tbs.mkv
```
SZ-log:
```
2018-02-14 13:32:30,136 (7f8f404d3700) :  INFO (logger:38) - subliminal_patch.core: Refining video with filebot
2018-02-14 13:32:32,862 (7f8f404d3700) :  DEBUG (logger:28) - subliminal_patch.refiners.common: Got original filename: the.alienist.s01e04.1080p.webrip.x264-tbs
```

[Logs_180214_13-42-12.zip](https://github.com/pannal/Sub-Zero.bundle/files/1724391/Logs_180214_13-42-12.zip)

**Comment by pannal** [comment]
Thanks, the issue was not filebot-exclusive, I've made the "filename crap removal" a little less excessive with my last revision. Will be fixed.

**Comment by pannal** [comment]
Fixed in 2.5.0.2241
